test/test_torch.py

```diff
@@ -6207,20 +6207,47 @@ def test_comparison_ops(self):
 
     def test_comparison_ops_must_take_bool_output(self):
         with self.assertRaisesRegex(RuntimeError, 'The output tensor of lt must be a bool'):
-            torch.lt(torch.tensor([True]), torch.tensor([False]), out=torch.empty(1, dtype=torch.uint8))
+            for op in [torch.lt, torch.le, torch.gt, torch.ge, torch.eq, torch.ne]:
+                op(torch.tensor([True]), torch.tensor([False]), out=torch.empty(1, dtype=torch.uint8))
 
     def test_inplace_comparison_ops_require_inputs_have_same_dtype(self):
         with self.assertRaisesRegex(RuntimeError, 'Expected object of scalar type'):
             torch.tensor([1], dtype=torch.int).lt_(torch.tensor([2], dtype=torch.long))
+            torch.tensor([1], dtype=torch.int).le_(torch.tensor([2], dtype=torch.long))
```

Comment:
Please clean up these tests by not repeating yourself. If we make this a for loop, it only needs to be 3 lines instead of 6